armi/reactor/converters/uniformMesh.py

```diff
@@ -673,7 +612,7 @@ def reset(self):
         self._cachedReactorCoreParamData = {}
         super().reset()
 
-    def _setParamsToUpdate(self):
+    def _setParamsToUpdate(self, direction):
         """Activate conversion of various paramters."""
         self.reactorParamNames = []
         self.blockParamNames = []
```

Comment:
Should this method instead be changed to raise a `NotImplemetedError`, since we never want people to actually instantiate this base converter class?

I'm not sure, your call.

Reply:
That's a good point. But theoretically if you're only interested in geometry and not parameters, it might be valid to instantiate the base class and use it?